src/components/_buttons/DepositAndWithdrawButton.tsx

```
@@ -88,7 +148,32 @@ export function DepositAndWithdrawButton({
+  // Debug logging for withdrawal button logic
+  if (
+    process.env.NEXT_PUBLIC_DEBUG_SORT === "1" &&
+    row.original?.name?.includes("Real Yield ETH")
+  ) {
+    console.log("Withdrawal button debug:", {
+      name: row.original?.name,
+      netValue: netValue,
+      coerceNetValue: coerceNetValue(netValue),
+      hasValueInVault: hasValueInVault,
+      isDeprecated: row.original?.deprecated,
+      buttonText: buttonText,
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)